src/frontend/src/pages/ProjectDetailPage/ProjectViewContainer/Project/ProjectDetailTabs.tsx

```diff
@@ -0,0 +1,51 @@
+/*
+ * This file is part of NER's FinishLine and licensed under GNU AGPLv3.
+ * See the LICENSE file in the repository root folder for details.
+ */
+
+import { Tab, Tabs } from '@mui/material';
+import { useEffect, useMemo, useState } from 'react';
+import { Link, useLocation, useRouteMatch } from 'react-router-dom';
+import { Project, wbsPipe } from 'shared';
+import { routes } from '../../../../utils/routes';
+
+interface ProjectDetailPageTabProps {
+  project: Project;
+}
+
+// Page block containing task list view
```

Comment:
this comment is not accurate

Reply:
Changed the comment to be relevant to the newly created tabs.